### ISSUE TITLE
attrs.multiple is string, isn't it?

### ISSUE THREAD

**Issue by kityan** [amitava82/angular-multiselect#79]
`isMultiple = attrs.multiple ? true : false` always true because attrs.multiple is string, no?
here: https://github.com/amitava82/angular-multiselect/blob/master/src/multiselect.js#L41

**Comment by amitava82** [comment]
You are right but then one would add `multiple` attribute for multi select and not add it for single select. However I think it can be enhanced to allow programmatic switching between single/multiselect.

**Comment by kityan** [comment]
Yes, definitely! I've not thinked about not adding attribute at all. That's because I use this method: if attribute can be ignored I write <element attr /> or <element attr="attr" />. But If <element attr="true"> than value has to be correctly transformed to Boolean.

Thanks for your directive. It was very helpful.
